pywbem/cim_obj.py

```diff
@@ -1702,13 +1731,25 @@ def property_list(self):
 
         This attribute is settable. For details, see the description of the
         same-named constructor parameter.
+
+        **Deprecated:** This attribute has been deprecated in pywbem v0.12.0.
+        Set only the desired properties on the object, instead of working with
+        this property filter.
         """
         return self._property_list
 
     @property_list.setter
     def property_list(self, property_list):
         """Setter method; for a description see the getter method."""
         if property_list is not None:
+            msg = "The 'property_list' init parameter and attribute of " \
+                "CIMInstance is deprecated; Set only the desired properties " \
```

Comment:
should we add 'or use the property list on the modify wbem operation to tell the server which properties are to be modified.